www-apps/gitea/gitea-1.8.2.ebuild

```diff
@@ -0,0 +1,85 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit golang-vcs-snapshot systemd user
+
+EGO_PN="code.gitea.io/gitea"
+KEYWORDS="~amd64 ~arm ~arm64"
+
+DESCRIPTION="A painless self-hosted Git service"
+HOMEPAGE="https://gitea.io"
+SRC_URI="https://github.com/go-gitea/gitea/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="pam sqlite"
+
+COMMON_DEPEND="pam? ( sys-libs/pam )"
+DEPEND="${COMMON_DEPEND}
+	dev-go/go-bindata"
+RDEPEND="${COMMON_DEPEND}
+	dev-vcs/git"
+
+DOCS=( custom/conf/app.ini.sample CONTRIBUTING.md README.md )
+S="${WORKDIR}/${P}/src/${EGO_PN}"
+
+pkg_setup() {
+	enewgroup git
+	enewuser git -1 /bin/bash /var/lib/gitea git
+}
+
+gitea_make() {
+	local my_tags=(
+		bindata
+		$(usev pam)
+		$(usex sqlite 'sqlite sqlite_unlock_notify' '')
+	)
+	local my_makeopt=(
+		DRONE_TAG=${PV}
+		TAGS="${my_tags[@]}"
+	)
+	GOPATH=${WORKDIR}/${P}:$(get_golibdir_gopath) emake "${my_makeopt[@]}" "$1"
+}
+
+src_compile() {
+	gitea_make generate
+	gitea_make build
+}
+
+src_test() {
+	gitea_make test
+}
+
+src_install() {
+	einstalldocs
+	dobin gitea
+	newconfd "${FILESDIR}"/gitea.confd-r1 gitea
+	newinitd "${FILESDIR}"/gitea.initd-r2 gitea
+	systemd_newunit "${FILESDIR}"/gitea.service-r2 gitea.service
+	diropts -m0750 -o git -g git
+	keepdir /etc/gitea
+	keepdir /var/lib/gitea /var/lib/gitea/custom /var/lib/gitea/data
+	keepdir /var/log/gitea
+}
+
+pkg_postinst() {
+	ewarn "The configuration path has been changed to ${EROOT}/etc/gitea/app.ini."
+	ewarn "Please adapt the gitea-repositories hooks and ssh authorized_keys."
+	ewarn "Depending on your configuration you should run something like:"
+	ewarn "sed -i -e 's#/var/lib/gitea/conf/app.ini#/etc/gitea/app.ini#' \\"
+	ewarn "  /var/lib/gitea/gitea-repositories/*/*/hooks/*/* \\"
+	ewarn "  /var/lib/gitea/.ssh/authorized_keys"
+
+	if [[ ! -e "${EROOT}/etc/gitea/app.ini" ]]; then
+		elog "No app.ini found, copying initial config over"
+		cp "${FILESDIR}"/app.ini "${EROOT}"/etc/gitea/ || die
```

Comment:
FILESDIR is not allowed in `pkg_*` phases. It doesn't exist in binpackages.

Reply:
Regarding this and the other changes, would it be fine to use the fixed and the *atomic* version of the this [ebuild](https://github.com/xdch47/gpo-xdch47/blob/master/www-apps/gitea/gitea-1.8.2-r1.ebuild) I created recently ??
Or does this Introduce to much changes / is unwanted ?

The background is that the service-files kept changing with every release and this might continue due to the systemd postgres slot issue (as discussed in the previous PR #11808 - see bug  https://bugs.gentoo.org/671596). From my point of view this way would be easier to maintain (diff changes), less *small* files in the portage tree and last but not least i like the *new helpers can read from stdin* feature, which I discovered recently on your blog, since it possible to prefixify those service files…

Reply:
Swelling the ebuild and disabling file reuse won't help. Why can't you install the file in src_install, and let etc-update take care of updates?

Reply:
we propsed that in #10961 - however it never got merged,
but the version of @hydrapolic got #11387, which followed the `minimal config` approach.

I the review's, I mentioned that - however `mininal config` way has been preferred - so I did not want to undo that

Reply:
> Swelling the ebuild and disabling file reuse won't help.

form me point of view it is quite small and more handy than splitting an already small file in 4 pieces, with have to be rev bump and removed separately (they got reset recently - still *-r2* is reached). (e.g. avoid something like in #11815)

it was just an suggestion -  it's fine if it does not suite your taste.